pinot-spi/src/main/java/org/apache/pinot/spi/utils/StringUtil.java

```diff
@@ -53,4 +57,22 @@ public static String sanitizeStringValue(String value, int maxLength) {
     }
     return value.substring(0, Math.min(index, maxLength));
   }
+
+  /**
+   * InputStream object to String
+   * @param inputStream InputStream to be converted
+   * @return String holding the contents of the inputStream
+   * @throws IOException
+   */
+  public static String inputStreamToString(InputStream inputStream)
```

Comment:
curious why you chose to add this vs using the short IOUtils.toString(InputStream, CharSet) method?

Reply:
Just wasn't aware commons dependency is already added to the module.
Changed.